utils/char_buffer_str.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2019 BiiLabs Co., Ltd. and Contributors
+ * All Rights Reserved.
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the MIT license. A copy of the license can be found in the file
+ * "LICENSE" at the root of this distribution.
+ */
+
+#ifndef CHAR_BUFFER_STR_H_
+#define CHAR_BUFFER_STR_H_
+
+#include <stdlib.h>
+#include <string.h>
+#include "common/ta_errors.h"
+#include "utils/char_buffer.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Store response message to json_result
+ * @param[in] res_buff Buffer of response message
+ * @param[out] json_result Response message
+ *
+ * @return
+ * - SC_OK on success
+ * - non-zero on error
+ */
+static inline status_t char_buffer_to_str(char_buffer_t* res_buff, char** json_result) {
```

Comment:
Use `strndup` instead?

Reply:
Do we really need `char_buffer_to_str`?
Is this what `strdup` or `strndup` do?

Reply:
To be honest, I agree with you. I think we coud replace `char_buffer_to_str` with `strdup` in other PR.